--- config.rb.orig	2020-09-23 10:27:08.624027794 -0400
+++ config.rb	2020-09-23 10:30:59.984333592 -0400
@@ -53,18 +53,18 @@
 ## command is running. When running as a daemon/service, this is put into a
 ## file in logs/archivesspace.out. You can change this file by changing the log
 ## value to a filepath that archivesspace has write access to.
-#AppConfig[:frontend_log] = "default"
+AppConfig[:frontend_log] = "/archivesspace/logs/archivesspace.out"
 ## Log level for the frontend, values: (everything) debug, info, warn, error, fatal (severe only)
-#AppConfig[:frontend_log_level] = "debug"
+AppConfig[:frontend_log_level] = "warn"
 ## Log level for the backend, values: (everything) debug, info, warn, error, fatal (severe only)
-#AppConfig[:backend_log] = "default"
-#AppConfig[:backend_log_level] = "debug"
+AppConfig[:backend_log] = "/archivesspace/logs/archivesspace.out"
+AppConfig[:backend_log_level] = "warn"
 #
-#AppConfig[:pui_log] = "default"
-#AppConfig[:pui_log_level] = "debug"
+AppConfig[:pui_log] = "/archivesspace/logs/archivesspace.out"
+AppConfig[:pui_log_level] = "info"
 #
-#AppConfig[:indexer_log] = "default"
-#AppConfig[:indexer_log_level] = "debug"
+AppConfig[:indexer_log] = "/archivesspace/logs/archivesspace.out"
+AppConfig[:indexer_log_level] = "info"
 #
 #
 ## Set to true to log all SQL statements.  Note that this will have a performance
